remoting/src/main/java/org/apache/rocketmq/remoting/protocol/RocketMQSerializable.java

```diff
@@ -243,15 +245,6 @@ public static HashMap<String, String> mapDeserialize(ByteBuf byteBuffer, int len
     }
 
     public static boolean isBlank(String str) {
-        int strLen;
-        if (str == null || (strLen = str.length()) == 0) {
-            return true;
-        }
-        for (int i = 0; i < strLen; i++) {
-            if (!Character.isWhitespace(str.charAt(i))) {
-                return false;
-            }
-        }
-        return true;
+        return StringUtils.isBlank(str);
```

Comment:
How to delete this method and replace all with StringUtils.isBlank?

Reply:
![image](https://user-images.githubusercontent.com/105986/207807717-e1ced284-fbf5-48cb-b0a3-a26b7025c62e.png)

I think this can be done ,this very method is only used in test

Reply:
fixed